app/components/primer/alpha/dialog/header.rb

```diff
@@ -16,6 +16,14 @@ class Header < Primer::Component
         }.freeze
         VARIANT_OPTIONS = VARIANT_MAPPINGS.keys
 
+        DEFAULT_CLOSE_SCHEME = :close
+        CLOSE_SCHEMES = [
```

Comment:
Schemes are usually something like `:danger`, `:primary` and so on.. Since you want to achieve a boolean flag to hide and show the close button, it maybe makes sense to make that a boolean like `show_close_button`.